src/current/v24.2/cluster-virtualization-metric-scopes.md

```diff
@@ -808,6 +933,7 @@ SERVER: n/a
 - `txn.restarts.writetoooldmulti`
 - `txn.rollbacks.async.failed`
 - `txn.rollbacks.failed`
+- `build.timestamp`
```

Comment:
nit. just wondering why `build.timestamp` is not in alphabetical order.

Reply:
I have no idea, I'll manually fix it.